oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestListContainer.jsx

```diff
@@ -19,12 +19,12 @@ export const RequestListContainer = ({ requestTypes, isLoading, loadingError, fe
 
   useEffect(() => {
     fetchRequests();
-  }, []);
+  }, [fetchRequests]);
```

Comment:
This effect is a duplicate of one in RecordsRequests component, won't requests be fetched twice unnecessarily?